### ISSUE TITLE
Limit which device to control

### ISSUE THREAD

**Issue by SimoMay** [gabe565/CastSponsorSkip#45]
### Feature request:
An env/flag to specify a device name (or multiple) to limit the control to only the specified devices.

**Comment by gabe565** [comment]
I agree that this functionality would be useful. I'm still working on this under #36, so closing this issue as a duplicate. Please see the linked issue for future updates!

**Comment by gabe565** [comment]
@SimoMay Quick heads up that v0.7.0 adds this! A device list can now be passed with the new `--devices` flag or with the `CSS_DEVICES` env.
